packages/grpc-native-core/index.d.ts

```diff
@@ -962,14 +968,14 @@ declare module "grpc" {
      * @param callback The callback to for when the response is received
      * @return An event emitter for stream related events
      */
-    makeClientStreamRequest(
+    makeClientStreamRequest<W>(
```

Comment:
The same concerns about conflating the request and response types apply here too.